desktop/apps/consign/routes.js

```diff
@@ -20,8 +20,8 @@ export const landingPage = async (req, res, next) => {
     inDemand.id = in_demand.set.id
 
     await inDemand.fetch({ cache: true })
-    const { ordered_set: recentlySold } = await metaphysics({ query: RecentlySoldQuery(recently_sold.set.id) })
-    const { sales } = await metaphysics({ query: SalesQuery() })
+    const { ordered_set: recentlySold } = await metaphysics({ query: RecentlySoldQuery(recently_sold.set.id), req })
+    const { sales } = await metaphysics({ query: SalesQuery(), req })
```

Comment:
Once we confirm that this approach is OK, I'll go ahead and update all of the routes to pass `req` here.